SimPEG/EM/Static/DC/FieldsDC.py

```diff
@@ -76,17 +76,28 @@ class Fields_CC(FieldsDC):
 
     def __init__(self, mesh, survey, **kwargs):
         FieldsDC.__init__(self, mesh, survey, **kwargs)
-        if self.mesh._meshType == "TREE":
-            if(self.prob.bc_type == 'Neumann'):
+
+        if getattr(self.survey.prob, 'bc_type', None) == 'Dirichlet':
+            self.cellGrad = -mesh.faceDiv.T
+        elif getattr(self.survey.prob, 'bc_type', None) == 'Neumann':
+            if self.mesh._meshType == "TREE":
                 raise NotImplementedError()
-            elif(self.prob.bc_type == 'Dirchlet'):
-                self.cellGrad = -mesh.faceDiv.T
+            mesh.setCellGradBC("neumann")
```

Comment:
Variable related cellGrad can be removed now. 
This was for computing electric field before.